bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -194,9 +198,19 @@ private HostedRepository getTargetRepo(PullRequestBot bot, String repoName, Prin
 
         var potentialTargetRepo = repoNameOptional.flatMap(forge::repository);
         if (potentialTargetRepo.isEmpty()) {
+            var branchNamesInCurrentRepo = bot.repo().branches().stream().map(HostedBranch::name).toList();
+            if (branchNamesInCurrentRepo.contains(repoName)) {
+                reply.println("There is a branch `" + repoName + "` in current repository `" + bot.repo().name() + "`.");
+                reply.println("To target a backport to this branch in current repository:");
+                reply.println("`/backport :" + repoName + "`");
+                reply.println();
+            }
             reply.println("The target repository `" + repoNameArg + "` is not a valid target for backports. ");
```

Comment:
I think this line should be first in the message, then followed by the branch message if there is a branch match.

Reply:
Sure, will fix it